### PR TITLE
Re-design desktop portfolio

### PR DESCRIPTION
**Pull request by kiboma2021**
Due to numerous changes suggested on week 2, and using the material provided by @[gtekle](https://github.com/gtekle) on RWD, I have re-disgned the website given that the chages looked important for my skills development (although the project was approved)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,880 +1,359 @@
...
+}
+
+article {
+  background-image: url("/imgs/background-desktop.png");
+  border-bottom-left-radius: 80px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (keroloussamy) [inline styles.css:60]: - [ ] Kindly your deployment is working very well just one thing is the background here is not appear, Kindly fix it.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @kiboma2021, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (kiboma2021) [reply styles.css]: @keroloussamy , The suggested improvement has been incorporated. Thank you

**Review by AnselemOdims — APPROVED** [review]
### Hi  @kiboma2021 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**